internal/rbd/rbd_util.go

```diff
@@ -600,36 +600,34 @@ func (rv *rbdVolume) getCloneDepth(ctx context.Context) (uint, error) {
 	vol.Monitors = rv.Monitors
 	vol.RbdImageName = rv.RbdImageName
 	vol.conn = rv.conn.Copy()
-
+	currVol := vol
```

Comment:
maybe assign `&vol` so that `currVol` becomes a `*rbdVolume` and you can assign `parentVol` directly to it?

Reply:
Sure